subprojects/logging/src/main/java/org/gradle/internal/logging/console/BuildStatusRenderer.java

```diff
@@ -63,6 +66,9 @@ public void onOutput(OutputEvent event) {
             // if it has no parent ID, assign this operation as the root operation
             if (startEvent.getParentId() == null && BUILD_PROGRESS_CATEGORY.equals(startEvent.getCategory())) {
                 rootOperationId = startEvent.getOperationId();
+                if (buildStartTimestamp == 0) {
+                    buildStartTimestamp = startEvent.getTimestamp();
```

Comment:
One consequence of this is that the timer may not match the build result. Testing this out I saw that the timer stopped at 26s but the build result said 34s. Is there a way we can reuse the `Clock` instance used in `BuildResultLogger`?